engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1886,12 +1886,21 @@ class _ViewConfiguration {
     this.gestureSettings = const GestureSettings(),
     this.displayFeatures = const <DisplayFeature>[],
     this.displayId = 0,
+    this.viewConstraints = const ViewConstraints(
+      minHeight: 0,
+      maxHeight: 0,
+      minWidth: 0,
+      maxWidth: 0,
+    ),
   });
 
   /// The identifier for a display for this view, in
   /// [PlatformDispatcher._displays].
   final int displayId;
 
+  /// The sizing constraints for this view.
```

Comment:
Let's call out that this is physical pixels since most of the Flutter framework assumes logical pixels.

```suggestion
  /// The sizing constraints for this view in physical pixels.
```